src/ServiceControl/Recoverability/Retries/RetryDocumentManager.cs

```diff
@@ -114,26 +114,33 @@ internal void AdoptOrphanedBatches(IDocumentSession session, out bool hasMoreWor
         {
             RavenQueryStatistics stats;
 
-            var orphanedBatchIds = session.Query<RetryBatch, RetryBatches_ByStatusAndSession>()
+            var orphanedBatches = session.Query<RetryBatch, RetryBatches_ByStatusAndSession>()
```

Comment:
Not sure how smart Raven is for this. Will this change what's going over the wire from a list of id's to a list of full documents?